client/my-sites/subscribers/components/subscribers-header/subscribers-header.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Gridicon } from '@automattic/components';
```

Comment:
`SubscribersHeader` currently only exists in main.tsx. I abstracted it to use in the new components. I'm not replacing it in main because we'll remove it when we take off the flag.